### DISCUSSION TITLE
Large Image Stutter

### DISCUSSION THREAD

**Discussion by ghost** [mienaiyami/yomikiru#25] [General]
**Please bear with me as I do not know anything about programming.**

Just read your comment on this [issue](https://github.com/mienaiyami/react-ts-offline-manga-reader/issues/6#issuecomment-1318789321). What I understood from reading the chromium jargon is that ,

1. The bug is at the side of chromium and you cannot do anything about it. Am I right?
2. The issue is caused by many decode requests which hit the cache limitation, right?
3. The setting now used in your application is to decode all the images in a folder.
4. Then rather than decoding all the images at once, will a setting of caching / decoding 5 images at once solve the problem?

**Comment by mienaiyami** [comment]
>1. The issue is at the side of chromium and you cannot do anything about it. Am I right?

Yes, its not present in Firefox or other non-chromium related browser.
>2. The issue is caused by many decode requests which hit the cache limitation, right?

Yes.
>Then rather than decoding all the images at once, will a setting of caching / decoding 5 images at once solve the problem?

You see, there's two different things.

1. Loading of raw images from your system.
2. Decoding of those loaded images which convert them to a format that can be displayed on screen.

For now, I load and decode all images at once, which worked perfectly fine for normal size images. 
But now I have been trying to decode just like you said, but this approach will cause stuttering on every 5 images, which might sound good for larger images, but it will also apply to normal sized images, making them stutter as well.

I was supposed to release update yesterday, but I didn't because I was in little busy, sorry about that. I will release new update soon with possibly less or no stutter.

**Comment by ghost** [comment]
> I was supposed to release update yesterday, but I didn't because I was in little busy, sorry about that.

- Take you time.

>I will release new update soon with possibly less or no stutter.

- You found the solution to the stutter that happened in normal sized images.
- You might find the solution to the present issue too. It might take time, but you could.

Thank you for your work.

Did the html mangareader application not help?

**Reply by mienaiyami** [reply]
> Did the html mangareader application not help?

Unfortunately i couldn't find any solution from it.

**Comment by ghost** [comment]


**Reply by ghost** [reply]
Never mind about the above suggestion.
Just searched online and found chromium to be best engine in page accuracy, faster loading speeds and many more.

**Reply by mienaiyami** [reply]
> If you use firefox's 'Quantum browser engine' instead of google chrome's 'chromium', wouldn't it solve the issue?

I could but that's not possible to do. It only supports chromium because of this [reason](https://www.quora.com/Why-did-GitHub-choose-the-Chromium-browser-instead-of-Firefox-Quantum-for-Electron-JS)

**Reply by mienaiyami** [reply]
You should not delete questions in imo as if someone else have same question they can find it here.

**Reply by ghost** [reply]
Oh okay. I thought since it was irrelevant it is no longer needed and deleted it.
